### PR TITLE
Kevin/new feature/pin password spending limits rebase

### PR DESCRIPTION
**Pull request by wolverineks**

### CODE REVIEW

File: src/locales/en_US.js [hunk 1]
```diff
@@ -98,6 +98,10 @@ const strings = {
   settings_button_use_touchID: 'Use TouchID',
   settings_days: 'Day(s)',
   settings_denominations_title: 'Denominations',
+  settings_options_title: 'Options',
+  settings_spending_limits: 'Spending Limits',
+  settings_password_spending_limit_title: 'Password Spending Limit',
+  settings_pin_spending_limit_title: 'Pin Spending Limit',
```
Comment (paullinator) [inline src/locales/en_US.js]: Pin should be all caps. "PIN"

File: src/modules/UI/scenes/Settings/CurrencySettings.ui.js [hunk 2]
```diff
@@ -9,10 +9,13 @@ import styles from './style'
 import {border as b} from '../../../utils'
 import Row from './components/Row.ui.js'
 import RadioRows from './components/RadioRows.ui.js'
+import RowRoute from './components/RowRoute.ui'
 
 import type {GuiDenomination} from '../../../../types'
 
 const SETTINGS_DENOMINATION_TEXT = s.strings.settings_denominations_title
+const SETTINGS_OPTIONS_TEXT      = s.strings.settings_options_title
```
Comment (paullinator) [inline src/modules/UI/scenes/Settings/CurrencySettings.ui.js]: These additional levels of indirection for strings is unnecessary. Please remove.

File: src/locales/en_US.js [hunk 3]
```diff
@@ -1,12 +1,28 @@
 // @flow
 
 const strings = {
+  accept_button_text: 'Accept',
```
Comment (paullinator) [inline src/locales/en_US.js]: Sorting these entries was unnecessary and a waste of time and makes reviewing harder. Please undo and add only necessary strings.

File: src/modules/UI/scenes/SpendingLimits/action.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+export const updateDailySpendingLimit = (currencyCode: string, isEnabled: boolean, dailySpendingLimit: string) => (dispatch: Dispatch, getState: GetState) => {
+  dispatch(updateDailySpendingLimitStart(currencyCode, isEnabled, dailySpendingLimit))
+
+  const state = getState()
+  const account = CORE_SELECTORS.getAccount(state)
+
+  return SETTINGS_API.setDailySpendingLimitRequest(account, currencyCode, isEnabled, dailySpendingLimit)
+  .then((settings) => {
+    console.log(settings)
```
Comment (paullinator) [inline src/modules/UI/scenes/SpendingLimits/action.js]: Please remove console.log for non Core routines and callbacks. Also make it a habit to not console.log full objects vs the individual relevant parameters

**Comment by paullinator** [comment]
Running this branch live, I've come across a couple other comments:

* Entering spending limits for any currency shows the toggle button as enabled for both daily spending and transaction spending limit
* Any attempt at typing in a password crashes with a red screen unhandled promise rejection

File: src/modules/UI/scenes/Settings/CurrencySettings.ui.js (hunk 2 above)
Comment (wolverineks) [reply src/modules/UI/scenes/Settings/CurrencySettings.ui.js]: I dont know what that means

**Comment by wolverineks** [comment]
> * Entering spending limits for any currency shows the toggle button as enabled for both daily spending and transaction spending limit

I don't understand this comment

> * Any attempt at typing in a password crashes with a red screen unhandled promise rejection

I'm unable to reproduce any crashes

**Comment by thehobbit85** [comment]
This PR has conflicts that needs to be resolved

**Pull request closed without merging**